src/daemon/daemon.c

```diff
@@ -1115,13 +1143,16 @@ static gboolean screensaver_active(GtkWidget* nw)
 	return active;
 }
 
+#ifdef HAVE_X11
 static gboolean fullscreen_window_exists(GtkWidget* nw)
 {
 	WnckScreen* wnck_screen;
 	WnckWorkspace* wnck_workspace;
 	GList* l;
 
-		wnck_screen = wnck_screen_get(GDK_SCREEN_XNUMBER(gdk_window_get_screen(gtk_widget_get_window(nw))));
+	g_return_val_if_fail (GDK_IS_X11_DISPLAY (gdk_display_get_default ()), FALSE);
+
+	wnck_screen = wnck_screen_get(GDK_SCREEN_XNUMBER(gdk_window_get_screen(gtk_widget_get_window(nw))));
```

Comment:
missing a space before opening bracket

Reply:
Is there a MATE style guide? Stuff like this seems pretty inconsistent. Mostly I try to match my new code to the surounding code, and not touch old code (this line, for example, I didn't change except for removing an indent, so I didn't reformat it).

Reply:
There are some tips on the wiki page [1], however they should be expanded/updated and what’s more, we should define it in a git hook for formatting the code to save headaches as other projects do.

https://github.com/mate-desktop/mate-wiki/blob/master/pages/code_style.md